packages/files-ui/src/Components/Modules/Settings/SubscriptionPlan/index.tsx

```diff
@@ -1,10 +1,35 @@
-import React from "react"
+import React, { useState } from "react"
 import CurrentProduct from "./CurrentProduct"
+import AddCardModal from "./AddCardModal"
+import { Button } from "@chainsafe/common-components"
+import { makeStyles, createStyles, ITheme } from "@chainsafe/common-theme"
 
+const useStyles = makeStyles(({ constants }: ITheme) =>
+  createStyles({
+    root: {
+      marginBottom: constants.generalUnit * 16
+    },
+    container: {
+      margin: constants.generalUnit * 4
+    }
+  })
+)
 
 const PlanView: React.FC = () => {
+  const classes = useStyles()
+
+  const [isAddCardModalOpen, setIsAddCardModalOpen ] = useState(false)
   return (
-    <CurrentProduct />
+    <div className={classes.root}>
+      <CurrentProduct />
+      <div className={classes.container}>
+        <Button onClick={() => setIsAddCardModalOpen(true)}>Add Card</Button>
```

Comment:
Translation is missing
```suggestion
        <Button onClick={() => setIsAddCardModalOpen(true)}>
          <Trans>Add Card</Trans>
        </Button>
```